2026 04 23 exec doc (#92)
sky-ecosystem/executive-votes: 2026/executive-vote-2026-04-23-msc-staking-rewards-update.md

---
title: Template - [Executive Vote] March 2026 Monthly Settlement Cycle, Update Staking Rewards, Adjust ALLOCATOR-BLOOM-A and ALLOCATOR-PATTERN-A DC-IAM Parameters, Whitelist Pattern ALMProxy, Prime Agent Proxy Spells - April 23, 2026

> summary: Execute Monthly Settlement Cycle and Treasury Management Function for March 2026, update LSSKY->SKY staking rewards, increase ALLOCATOR-BLOOM-A `gap`; increase ALLOCATOR-PATTERN-A `gap` and `line`, whitelist Pattern ALMProxy on the LitePSM, whitelist proxy spells for Spark, Grove, and Pattern.

Reviewer: The voting portal does not support inline code in titles and summaries

```suggestion
summary: Execute Monthly Settlement Cycle and Treasury Management Function for March 2026, update LSSKY->SKY staking rewards, increase ALLOCATOR-BLOOM-A target available debt; increase ALLOCATOR-PATTERN-A target available debt and maximum debt ceiling, whitelist Pattern ALMProxy on the LitePSM, whitelist proxy spells for Spark, Grove, and Pattern.
```

---

## Executive Summary

If this executive proposal passes, the following **actions** will occur within the Sky Protocol:

- The Monthly Settlement Cycle and Treasury Management Function for March 2026 will be executed.
- LSSKY->SKY staking rewards will be updated.
- The [Debt Ceiling Instant Access Module (DC-IAM) parameters](https://sky-atlas.io/#93c9f662-4e0d-477e-8fc9-e3726877e842) for ALLOCATOR-BLOOM-A and ALLOCATOR-PATTERN-A will be adjusted.
- Pattern ALMProxy will be whitelisted on the LitePSM.
- Prime Agent proxy spells for Spark, Grove, and Pattern will be whitelisted in their respective StarGuard modules.

**Voting for this executive proposal will place your SKY in support of the actions outlined above.**

Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://sky-atlas.io/#A.1.9.3.1). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Sky Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to [**24 hours**](https://sky-atlas.io/#A.1.9.3.1.2).

> This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.

Reviewer: We probably no longer *need* office hours for the core spell. Adding for info only, suspect primes would prefer it and possibly factored in core office hours when coding.

Author: Agreed, it was likely too late to change course because of this potential assumption from the primes.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Sky Protocol.

---

## Proposal Details

### Monthly Settlement Cycle for March 2026

- **Authorization**: [Atlas A.2.4 - Sky Core Monthly Settlement Cycle](https://sky-atlas.io/#A.2.4)
- **Proposal**: [Forum Post](https://forum.skyeco.com/t/msc-7-settlement-summary-march-2026/27844)

If this executive proposal passes, then the March 2026 Monthly Settlement Cycle will be executed by taking the following steps.

#### Spark

- Mint **7,662,339 USDS** debt in ALLOCATOR-SPARK-A and transfer the amount to the Surplus Buffer.
- Send **1,725,726 USDS** from the Surplus Buffer to the [SPARK_SUBPROXY](https://etherscan.io/address/0x3300f198988e4C9C63F75dF86De36421f06af8c4).

#### Grove

- Mint **6,290,684 USDS** debt in ALLOCATOR-BLOOM-A and transfer the amount to the Surplus Buffer.
- Send **138,412 USDS** from the Surplus Buffer to the [GROVE_SUBPROXY](https://etherscan.io/address/0x1369f7b2b38c76B6478c0f0E66D94923421891Ba).

#### Keel

> - Send **30,241 USDS** from the surplus buffer to the [KEEL_SUBPROXY](https://etherscan.io/address/0x355cd90ecb1b409fdf8b64c4473c3b858da2c310).

Reviewer: checksum address

```suggestion
- Send **30,241 USDS** from the surplus buffer to the [KEEL_SUBPROXY](https://etherscan.io/address/0x355CD90Ecb1b409Fdf8b64c4473C3B858dA2c310).
```

#### Treasury Management Function

- Transfer **678,176 USDS** from the Surplus Buffer to the [Core Council Buffer](https://etherscan.io/address/0x210CFcF53d1f9648C1c4dcaEE677f0Cb06914364).
- Transfer **33,908 USDS** from the Surplus Buffer to the [Aligned Delegates Buffer](https://etherscan.io/address/0x37FC5d447c8c54326C62b697f674c93eaD2A93A3).

### LSSKY->SKY Staking Rewards Normalization

- **Authorization**: [Atlas A.2.3.1.4.2.2.2 - SKY Staking Rewards](https://sky-atlas.io/#293e4c9f-1e26-4d4b-b769-650a02eca8b8), [Core Facilitator Authorization](https://forum.skyeco.com/t/lssky-to-sky-rewards-sky-rewards-for-sky-stakers-normalization-configuration/27721/15)
- **Proposal**: [Forum Post](https://forum.skyeco.com/t/lssky-to-sky-rewards-sky-rewards-for-sky-stakers-normalization-configuration/27721/14)

If this executive proposal passes, then the distribution of SKY token rewards to SKY stakers will be normalized as per [A.4.4.1.4.2 - Short Term SKY Rewards For SKY Stakers](https://sky-atlas.io/#A.4.4.1.4.2), by executing the following actions:

> - Update LSSKY->SKY Farm vest by calling `TreasuryFundedFarmingInit.updateFarmVest()` with params:

Reviewer: add link to the function code

```suggestion
- Update LSSKY->SKY Farm vest by calling [`TreasuryFundedFarmingInit.updateFarmVest()`](https://github.com/sky-ecosystem/endgame-toolkit/blob/master/script/dependencies/treasury-funded-farms/TreasuryFundedFarmingInit.sol#L128) with params:
```

- **Proposal**: [Forum Post](https://forum.skyeco.com/t/april-23-2026-proposed-changes-to-spark-for-upcoming-spell/27831)

The following grant transfers will be made:

- Spark Foundation Grant
  - Recipient address: [0x92e4629a4510AF5819d7D1601464C233599fF5ec](https://etherscan.io/address/0x92e4629a4510AF5819d7D1601464C233599fF5ec).
  - Transfer amount: **1,100,000 USDS**.
- Spark Asset Foundation Grant
  - Recipient address: [0xEabCb8C0346Ac072437362f1692706BA5768A911](https://etherscan.io/address/0xEabCb8C0346Ac072437362f1692706BA5768A911).
  - Transfer amount: **100,000 USDS**.

#### Grove

##### [Ethereum] Increase USDS Mint Rate Limits

- **Authorization**: [Governance Poll 1628](https://vote.sky.money/polling/QmVAKhR6)
- **Proposal**: [Forum Post](https://forum.skyeco.com/t/april-23-2026-proposed-changes-to-grove-for-upcoming-spell/27829)

The following [rate limit](https://sky-atlas.io/#A.2.2.9.1.1.1.2.2) changes will be made to the Grove Liquidity Layer:

- USDS Mint (LIMIT_USDS_MINT):
  - Max amount: **500 million USDS**.
  - Slope: **500 million USDS per day**.

> 
> 
> ##### [Ethereum] Onboard USDS to Centrifuge JTRSY

Reviewer: ```suggestion
  - Slope: **500 million USDS per day**.

##### [Ethereum] Onboard USDS to Centrifuge JTRSY
```

- CCTP USDC (Avalanche → Ethereum):
  - Max amount: **Unlimited**.
  - Slope: **Unlimited**.

##### [Avalanche] Onboard USDS/USDC Curve Stableswap Swaps & LP

- **Authorization**: [Governance Poll 1628](https://vote.sky.money/polling/QmVAKhR6)
- **Proposal**: [Forum Post](https://forum.skyeco.com/t/april-23-2026-proposed-changes-to-grove-for-upcoming-spell/27829)

USDS/USDC Curve Stableswap Swaps and LP will onboarded with the following [rate limits](https://sky-atlas.io/#A.2.2.9.1.1.1.2.2):

- Pool: [0xA9d7d3D7e68a0cae89FB33c736199172f405C8D3](https://snowscan.xyz/address/0xA9d7d3D7e68a0cae89FB33c736199172f405C8D3)
- Swaps:
  - **5 million USDS max**
  - **100 million USDS per day slope**
  - **0.1% maxSlippage**
- LP Deposits
  - **50 million USDS max**
  - **50 million USDS per day slope**
- LP Withdrawals
  - **Unlimited**

#### Pattern

##### Pattern Liquidity Layer - Ethereum Mainnet Deployment

- **Authorization**: [Governance Poll 1628](https://vote.sky.money/polling/QmVAKhR6)
- **Proposal**: [Forum Post](https://forum.skyeco.com/t/proposed-changes-to-pattern-for-upcoming-spell/27835)

> The Pattern Liquidity Layer will be deployed on mainnet with the contracts described in this [forum post](https://forum.skyeco.com/t/proposed-changes-to-pattern-for-upcoming-spell/27835).

Reviewer: List the contracts explicitly, use format we used for Obex: https://github.com/sky-ecosystem/executive-votes/blob/main/2025/executive-vote-2025-11-13-solana-bridge-migration.md#mainnet-initialize-the-obex-liquidity-layer

##### Pattern Onboard Maple syrupUSDS - Ethereum Mainnet Deployment

- **Authorization**: [Governance Poll 1628](https://vote.sky.money/polling/QmVAKhR6)
- **Proposal**: [Forum Post](https://forum.skyeco.com/t/proposed-changes-to-pattern-for-upcoming-spell/27835)

syrupUSDS will be onboarded to the Pattern Liquidity Layer with the following [rate limits]((https://sky-atlas.io/#A.2.2.9.1.1.1.2.2)):

- Max amount: **100 million USDC**.
- Slope: **20 million USDC per day**.

> - Notes: Adheres to the RRC constraints. Withdrawals only.

Reviewer: syrupUSDC not syrupUSDS

- **Authorization**: [Governance Poll 1628](https://vote.sky.money/polling/QmVAKhR6)
- **Proposal**: [Forum Post](https://forum.skyeco.com/t/april-23-2026-proposed-changes-to-grove-for-upcoming-spell/27829)

The CCTP USDC transfer [rate limits](https://sky-atlas.io/#A.2.2.9.1.1.1.2.2) from Avalanche to Ethereum will be updated as follows:

- CCTP USDC (Avalanche → Ethereum):
  - Max amount: **Unlimited**.
  - Slope: **Unlimited**.

##### [Avalanche] Onboard USDS/USDC Curve Stableswap Swaps & LP

- **Authorization**: [Governance Poll 1628](https://vote.sky.money/polling/QmVAKhR6)
- **Proposal**: [Forum Post](https://forum.skyeco.com/t/april-23-2026-proposed-changes-to-grove-for-upcoming-spell/27829)

USDS/USDC Curve Stableswap Swaps and LP will onboarded with the following [rate limits](https://sky-atlas.io/#A.2.2.9.1.1.1.2.2):

- Pool: [0xA9d7d3D7e68a0cae89FB33c736199172f405C8D3](https://snowscan.xyz/address/0xA9d7d3D7e68a0cae89FB33c736199172f405C8D3)
- Swaps:
  - **5 million USDS max**
  - **100 million USDS per day slope**
  - **0.1% maxSlippage**
- LP Deposits
  - **50 million USDS max**
  - **50 million USDS per day slope**
- LP Withdrawals
  - **Unlimited**

#### Pattern

> ##### Pattern Liquidity Layer - Ethereum Mainnet Deployment

Reviewer: make consistent with other Primes

```suggestion
##### [Ethereum] Deploy Pattern Liquidity Layer
```

| ALM Proxy            | [0xbA43325E91C79E500486a23E953ab3d8C46f169F](https://etherscan.io/address/0xbA43325E91C79E500486a23E953ab3d8C46f169F) | Pattern
| Rate Limits          | [0xa77f69f90646A4c0e44cEe1D44Fab08bEb4EA204](https://etherscan.io/address/0xa77f69f90646A4c0e44cEe1D44Fab08bEb4EA204) | Pattern
| Relayer          | [0xd00665Df77E0b1294Ae2bdC3662F870092f6737B](https://etherscan.io/address/0xd00665Df77E0b1294Ae2bdC3662F870092f6737B) | Soter (GovOps)
| Freezer          | [0xe728D67bca6cb18dE249325792b6379Eef4618bB](https://etherscan.io/address/0xe728D67bca6cb18dE249325792b6379Eef4618bB) | Soter (GovOps)

###### Initial Parameters

Pattern Liquidity Layer [rate limits](https://sky-atlas.io/#A.2.2.9.1.1.1.2.2) for initial activation:

| Parameter |            Max Amount          |   Slope (Tokens/Day)    |
|----------|------------|-------------------|
|LIMIT_USDS_MINT        |100,000,000      |	50,000,000   |
|LIMIT_USDS_TO_USDC	    |100,000,000	  |   50,000,000 |
|LIMIT_4626_DEPOSIT (Maple Finance)       |100,000,000   |  20,000,000 |
| LIMIT_MAPLE_REDEEM    | type(uint256).max | 0 |

> ##### Pattern Onboard Maple syrupUSDC - Ethereum Mainnet Deployment

Reviewer: make consistent with other Primes

```suggestion
##### [Ethereum] Onboard Maple syrupUSDC
```
